README.md

```diff
@@ -118,41 +126,76 @@ This uses a standard `rspec` and `rubocop` stack. To run these locally:
 
 ```bash
 bin/rake
+# or
+bundle exec rspec
+```
+
+### JavaScript
+
+```bash
+npm test
 ```
 
 ---
 
 ## Troubleshooting
 
-_I see Page Not Found when I log in and try to create a job listing_
+* I see Page Not Found when I log in and try to create a job listing.
 
-Try importing the school data if you have not already. When your sign in account was created, it was assigned to a
-school via a URN, and you may not have a school in your database with the same URN.
+Try importing the school data if you have not already. When your sign in account was created, it was assigned to a school via a URN, and you may not have a school in your database with the same URN.
 
 ---
 
-## Dependencies
+## Setup
+
+Welcome! :tada: :fireworks:
+
+As you are on-boarded, please make a note of any issues you come across, and update the [documentation](/documentation/new-joiners.md), to improve the process for the next person.
 
-### Baseline
+First, clone the project:
 
 ```bash
-Ruby 2.6.6
+git clone git@github.com:DFE-Digital/teacher-vacancy-service.git
+```
+
+If you are on a new device, remember to [generate a new SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+
+### Dependencies
+
+If you are on a Mac, you most likely have both PostgreSQL and rbenv (a Ruby version manager) installed already.
+
+#### Ruby version
+
+```bash
+$ ruby --version
+
+ruby 2.6.6
 ```
 
-### Services
+The source of truth for the Ruby version is the [Gemfile](/Gemfile#L3).
+
+If using rbenv:
+
+```bash
+rbenv install 2.6.6
+```
+
+If you are using rbenv and you get the message `Your Ruby version is 2.6.3, but your Gemfile specified 2.6.6`, you may need to do [this](https://stackoverflow.com/questions/10940736/rbenv-not-changing-ruby-version/12150580#12150580), and then reload your terminal.
+
+#### Services
 
 Make sure you have the following services configured and running on your development background:
 
- * [Postgresql](https://postgresql.org)
- * [Redis](https://redis.io)
+ * [PostgreSQL](https://postgresql.org)
+ * [Redis](https://redis.io/topics/quickstart)
```

Comment:
might be worth mentioning about starting the redis server outside of the project folder

Reply:
actually probably not needed with gitignore chnages